main.go

```diff
@@ -1,9 +1,26 @@
 package main
 
 import (
+	"flag"
 	"fmt"
+	"github.com/LinuxAtApp/matterleast/servercom"
 )
-
+/*
+Main usage: `go run main.go -u <username> -p <password> <url>
+Package demontrates simple login functions using the servercom package's methods.
+*/
 func main() {
 	fmt.Println("matterleast")
-}
+	username := flag.String("u", "", "Username")
+	password := flag.String("p", "", "Password")
+	flag.Parse()
+	url := flag.Arg(0)
+	//Creates client and logs user in.
+	client:= *servercom.Startup(url, *username, *password)
+	//Tests if login was successful. 
+	if servercom.Connected(client) {
```

Comment:
Specifically, try struct embedding. It would look something like this:
```go
type Server struct {
    *mm.Client
}
```
Let's say that we had a `*servercom.Server` called `s`. We could access the client with `s.Client`. However, we could also call custom methods like `s.Connected()`.